src/bootstrap-table.js

```diff
@@ -685,8 +685,10 @@ class BootstrapTable {
         return
       }
 
-      this.searchText = text
-      this.options.searchText = text
+      if ($(currentTarget).hasClass('search-input')) {
```

Comment:
this.options.search is not an option here?

Reply:
No is not an option, because i have to check if the input which triggered the search is an input for the filter-control extension or the normal search input.
With `this.options.search` i only know if the search is enabled or not.